src/deepsparse/yolact/schemas.py

```diff
@@ -40,7 +41,7 @@ class YOLACTInputSchema(BaseModel, Splittable):
     Input Model for YOLACT
     """
 
-    images: Union[str, numpy.ndarray, List[Union[str, numpy.ndarray]]] = Field(
+    images: Union[str, List[str], List[Any]] = Field(
```

Comment:
Can you capture what `Any` is intended to cover either via a comment or adding to description?

Reply:
Done!